src/LRS.js

```diff
@@ -747,7 +747,12 @@ TinCan client library
 
             for (i = 0; i < idProps.length; i += 1) {
                 if (typeof cfg.params[idProps[i]] !== "undefined") {
-                    params[idProps[i]] = cfg.params[idProps[i]].id;
+                    if (typeof cfg.params[idProps[i]] === "string") {
+                        params[idProps[i]] = cfg.params[idProps[i]];
+                    }
+                    else {
+                        params[idProps[i]] = cfg.params[idProps[i]].id;
+                    }
```

Comment:
@brianjmiller Checking if `id` is undefined works, but does it make sense to check explicitly for `TinCan.Verb` or `TinCan.Activity` after that? Or at that point, if the object has an id, should we just let it go through even if that object isn't a Verb or Activity?